initfiles/bash/etc/init.d/install-init.in

```diff
@@ -289,9 +289,13 @@ if [ "${DISTRIB_NAME}" = "ubuntu" ] &&
    dpkg -l | grep -q -e "^ii[[:space:]]*systemd "
    if [ $? -eq 0 ]; then
       cp ${INSTALL_DIR}/etc/init.d/hpcc-init.service /lib/systemd/system/
+      chmod 644 /lib/systemd/system/hpcc-init.service
```

Comment:
ok, I agree a less involved change to chmod the original files.  I will update and re-push.